<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Item</class>
 <widget class="QWidget" name="Item">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="index_label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ip_label">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="mac_label">
     <property name="text">
      <string>aa:bb:cc:dd</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="download_bar">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
